src/js/components/FieldWrapper/FieldWrapper.js

```diff
@@ -40,6 +41,10 @@ class FieldWrapper extends PureComponent {
     }
 
     const FieldTag = (FieldTypes[field] || Input)
+ 
+    if ((field == 'checkbox' || field == 'radio') && fieldProps.options) {
```

Comment:
It's weird that this is single-quotes but your formatter looks like it converted all other single quotes to doubles. Might want to run your formatter against this file?